ydb/public/lib/ydb_cli/commands/ydb_dynamic_config.cpp

```diff
@@ -48,6 +73,7 @@ TCommandConfig::TCommandConfig(
     AddCommand(std::make_unique<TCommandConfigReplace>(allowEmptyDatabase));
     AddCommand(std::make_unique<TCommandConfigResolve>());
     AddCommand(std::make_unique<TCommandGenerateDynamicConfig>(allowEmptyDatabase));
+    AddCommand(std::make_unique<TCommandConfigInit>());
```

Comment:
It will add command to all subtrees, like `node config`, `cluster config` and `database config`. Make separate `TNodeConfig` for this command.